nncf/common/graph/layer_attributes.py

```diff
@@ -125,6 +125,15 @@ def get_target_dim_for_compression(self) -> int:
         return 0
 
 
+class MatmulAttributes(BaseLayerAttributes):
```

Comment:
Introduced new class for `matmul` specifically, since `LinearLayerAttrbiutes` was designed for modules with weights and biases.